### DISCUSSION TITLE
Music Generation Models: Which dataset was used?

### DISCUSSION THREAD

**Discussion by candidexmedia** [intel/openvino-plugins-ai-audacity#308] [Q&A] [answered]
Hi OpenVINO team! I'm curious to know how the musicgen models were trained, what dataset was used, and how the data was sourced.

**Comment by RyanMetcalfeInt8** [comment, marked as answer]
Hello! The Music Generation features uses the MusicGen-small model trained by Meta, and released publicly here: https://huggingface.co/facebook/musicgen-small

Check the model card there for more information about the method(s) used, training set, etc. Thanks!

**Reply by candidexmedia** [reply]
Thank you so much for your quick reply and link!

In case anyone else is wondering:

> The model was trained on licensed data using the following sources: the [Meta Music Initiative Sound Collection](https://www.fb.com/sound), [Shutterstock music collection](https://www.shutterstock.com/music) and the [Pond5 music collection](https://www.pond5.com/). See the paper for more details about the training set and corresponding preprocessing.

**Comment by MarcoRavich** [comment]
Bump.

A new - that seems better than others - model is on the scene:
Git [HeartMuLa](https://github.com/HeartMuLa/heartlib#readme)

HF (models): https://huggingface.co/HeartMuLa

Thanks to @F1shYi for.
